bindings/matrix-sdk-crypto-js/src/machine.rs

```diff
@@ -763,6 +767,155 @@ impl OlmMachine {
         }))
     }
 
+    /// Store the recovery key in the crypto store.
```

Comment:
IMHO we should not use the term "recovery key" here: https://github.com/matrix-org/matrix-spec/issues/1571